src/bridge/bridge.c

```diff
@@ -698,6 +696,27 @@ main (int argc,
   signal (SIGABRT, cockpit_test_signal_backtrace);
   signal (SIGSEGV, cockpit_test_signal_backtrace);
 
+  /* In case we are run via sshd and we have journald, make sure all
+   * logging output ends up in the journal on *this* machine, not sent
+   * back to the client.
+   */
+  if (g_getenv ("SSH_CONNECTION") && !g_log_writer_is_journald (2) && ! isatty(2))
+    {
+      int fd = sd_journal_stream_fd ("cockpit ssh session stderr", LOG_WARNING, 0);
```

Comment:
I used `LOG_WARNING` because it's what was there before.  `stderr` should be for errors, I guess the thinking goes?

`cockpit-ssh` is not a good identifier because it would suggest that the messages come from `cockpit-ssh`.  I could imagine dropping `stderr` to get "cockpit ssh session", or maybe even "cockpit session", but I want to make it clear that this is not the same of a binary.  I spent a lot of wasted time trying to track problems down because output that was coming from `cockpit-ssh` was being labelled `cockpit-bridge`.

Reply:
As discussed on IRC, let's use `cockpit/ssh` as identifier. Agreeing about `LOG_WARNING`.